service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -46,14 +49,18 @@ protected AbstractEmailContentProvider(String uiBaseUrl, DateFormatterService da
             .put("dataPresent", !ordersAndDirections.isEmpty() ? "Yes" : "No")
             .put("fullStop", !ordersAndDirections.isEmpty() ? "No" : "Yes")
             .put("timeFramePresent", timeFrame.isPresent() ? "Yes" : "No")
-            .put("timeFrameValue", timeFrame.orElse(""))
+            .put("timeFrameValue", uncapitalize(timeFrame.orElse("")))
+            .put("urgentHearing", timeFrame.isPresent() && timeFrame.get().equals("Same day") ? "Yes" : "No")
+            .put("nonUrgentHearing", timeFrame.isPresent() && !timeFrame.get().equals("Same day") ? "Yes" : "No")
+            .put("firstRespondentName", getFirstRespondentLastName(caseData))
```

Comment:
would it be beneficial to move these three into a separate method? So we could have something like 
```java
Map<String, Object> getPartyPersonalisationBuilder(CaseData caseData) {
        Optional<String> timeFrame = Optional.ofNullable(caseData.getHearing().getTimeFrame());

        return ImmutableMap.<String, Object>builder()
            .put("urgentHearing", timeFrame.isPresent() && timeFrame.get().equals("Same day") ? "Yes" : "No")
            .put("nonUrgentHearing", timeFrame.isPresent() && !timeFrame.get().equals("Same day") ? "Yes" : "No")
            .put("firstRespondentName", getFirstRespondentLastName(caseData))
            .build();
    }
```

With this we do not need to update existing method and we have separation of concerns. Thoughts?

Reply:
Don't think it is necessary - it's fine to add these to getCasePersonalisationBuilder, this method is only used in places where we generate notifications for submitting application (and in those places we want these new values). So don't need to separate into another method